pkgs/servers/caddy/default.nix

```
@@ -35,7 +42,34 @@ buildGoModule {
     "-X github.com/caddyserver/caddy/v2.CustomVersion=${version}"
   ];
 
-  nativeBuildInputs = [ installShellFiles ];
+  nativeBuildInputs = [ gnused installShellFiles ];
+
+  modBuildPhase = ''
+    for module in ${builtins.toString (attrsToModules externalPlugins)}; do
+      sed -i "/standard/a _ \"$module\"" ./cmd/caddy/main.go
+    done
```

We could probably do without `sed` here and further below. Something like this maybe?

```suggestion
    substituteInPlace ./cmd/caddy/main.go \
      --replace "  // plug in Caddy modules here" "${
        lib.concatMapStringsSep "\\n"
          (module: "  _ ${module}")
          (attrsToModules externalPlugins)
      }"
```